php/plan.sh

```diff
@@ -1,15 +1,15 @@
 pkg_name=php
 pkg_distname=php
```

Comment:
Seems we should remove the pkg_distname if it matches the pkg_name and we are no longer using it any more.